manual/gce-sru-19.4.33.txt

```diff
@@ -96,3 +96,1954 @@ done
+
+Starting stage: modules-config
+|`->config-emit_upstart ran successfully @11.52100s +00.00000s
+|`->config-snap ran successfully @11.52100s +00.00100s
+|`->config-snap_config ran successfully @11.52200s +00.00100s
+|`->config-ssh-import-id ran successfully @11.52300s +01.08400s
+|`->config-locale ran successfully @12.60800s +00.00100s
+|`->config-set-passwords ran successfully @12.60900s +00.00100s
+|`->config-grub-dpkg ran successfully @12.61000s +00.34100s
+|`->config-apt-pipelining ran successfully @12.95100s +00.00100s
+|`->config-apt-configure ran successfully @12.95200s +00.17100s
+|`->config-ubuntu-advantage ran successfully @13.12300s +00.00100s
+|`->config-ntp ran successfully @13.12400s +00.08600s
+|`->config-timezone ran successfully @13.21200s +00.00100s
+|`->config-disable-ec2-metadata ran successfully @13.21300s +00.00000s
+|`->config-runcmd ran successfully @13.21300s +00.00100s
+|`->config-byobu ran successfully @13.21400s +00.00100s
+Finished stage: (modules-config) 01.72300 seconds
+
+Starting stage: modules-final
+|`->config-snappy ran successfully @13.72200s +00.00200s
+|`->config-package-update-upgrade-install ran successfully @13.72400s +00.00100s
+|`->config-fan ran successfully @13.72500s +00.00000s
+|`->config-landscape ran successfully @13.72600s +00.00000s
+|`->config-lxd ran successfully @13.72600s +00.00100s
+|`->config-ubuntu-drivers ran successfully @13.72700s +00.00000s
+|`->config-puppet ran successfully @13.72800s +00.00000s
+|`->config-chef ran successfully @13.72800s +00.00100s
+|`->config-mcollective ran successfully @13.72900s +00.00100s
+|`->config-salt-minion ran successfully @13.73000s +00.00000s
+|`->config-rightscale_userdata ran successfully @13.73000s +00.00100s
+|`->config-scripts-vendor ran successfully @13.73100s +00.00100s
+|`->config-scripts-per-once ran successfully @13.73200s +00.00000s
+|`->config-scripts-per-boot ran successfully @13.73200s +00.00100s
+|`->config-scripts-per-instance ran successfully @13.73300s +00.00000s
+|`->config-scripts-user ran successfully @13.73400s +00.00000s
+|`->config-ssh-authkey-fingerprints ran successfully @13.73400s +00.00300s
+|`->config-keys-to-console ran successfully @13.73700s +00.02400s
+|`->config-phone-home ran successfully @13.76200s +00.00100s
+|`->config-final-message ran successfully @13.76300s +00.00400s
+|`->config-power-state-change ran successfully @13.76800s +00.00000s
+Finished stage: (modules-final) 00.07900 seconds
+
+Total Time: 2.81700 seconds
+
+1 boot records analyzed
+-- Boot Record 01 --
+     01.08400s (modules-config/config-ssh-import-id)
+     00.34100s (modules-config/config-grub-dpkg)
+     00.30800s (init-network/config-ssh)
+     00.17100s (modules-config/config-apt-configure)
+     00.08600s (modules-config/config-ntp)
+     00.08100s (init-network/config-growpart)
+     00.07000s (init-network/config-resizefs)
+     00.06600s (init-network/config-users-groups)
+     00.04600s (init-network/searc
```

Comment:
Doesn't matter for this PR, but GCE should only update metadata per new instance. This text is misleading.
Might want a new PR to fix future GCE SRU validation in sru-templates/manual/gce to assert that 'New instance first  boot' is the only thing that gets logged.

Reply:
man github highlighted the wrong part of the big log file again.
We should check that only 'New instance first boot' shows up not 'System boot' in /var/log/cloud-init.log